pages/national_resources/se_resources.md

```diff
@@ -97,22 +97,22 @@ If personal data is processed in your research, contact your institute’s Data
 
 The [**SciLifeLab Data Centre**](https://www.scilifelab.se/data)  provides services for IT and data management, including Data Stewardship Wizard instance (for writing data management plans), the Swedish Pathogens Portal, and the SciLifeLab Data Repository. 
 
-Data stewards at [**NBIS**](https://nbis.se/) (ELIXIR-SE) provide consultation and support services regarding data management questions, including e.g. guidance when writing data management plans and when doing submissions to domain-specific repositories. For information about this and other resources at NBIS please see the [**Data Management**](https://nbis.se/services/guidance-on-data-handling) page. An upcoming resource is the [**FEGA Sweden**](https://fega.nbis.se/), a secure data archive and sharing platform for sensitive datasets, which will be integrated with the {% tool "the-european-genome-phenome-archive" %}.
+Data stewards at [**NBIS**](https://nbis.se/) (ELIXIR-SE) provide consultation and support services regarding data management questions, including e.g. guidance when writing data management plans and when doing submissions to domain-specific repositories. For information about this and other resources at NBIS please see the [**Data Management**](https://nbis.se/services/guidance-on-data-management) page. An upcoming resource is the [**FEGA Sweden**](https://fega.nbis.se/), a secure data archive and sharing platform for sensitive datasets, which will be integrated with the {% tool "the-european-genome-phenome-archive" %}.
```

Comment:
is FEGA Sweden still an upcoming resource? as it's already launched, as far as I understand :)

Reply:
```suggestion
Data stewards at [**NBIS**](https://nbis.se/) (ELIXIR-SE) provide consultation and support services regarding data management questions, including e.g. guidance when writing data management plans and when doing submissions to domain-specific repositories. For information about this and other resources at NBIS please see the [**Data Management**](https://nbis.se/services/guidance-on-data-management) page. An important resource is the [**FEGA Sweden**](https://fega.nbis.se/), a secure data archive and sharing platform for sensitive datasets, which will be integrated with the {% tool "the-european-genome-phenome-archive" %}.
```